eclass/qt5-build.eclass

```diff
@@ -670,6 +673,33 @@ qt5_base_configure() {
 
 }
 
+# @FUNCTION: qt5_tools_configure
+# @INTERNAL
+# @DESCRIPTION:
+# Disables modules other than ${PN} belonging to qttools.
+qt5_tools_configure() {
+	# configure arguments
+	local qmakeargs=(
+		--
+		# not packaged in Gentoo
+		-no-feature-distancefieldgenerator
+		-no-feature-kmap2qmap
+		-no-feature-macdeployqt
+		-no-feature-makeqpf
+		-no-feature-qev
+		-no-feature-qtattributionsscanner
+		-no-feature-windeployqt
+		-no-feature-winrtrunner
+	)
+
+	for x in assistant designer linguist pixeltool qdbus qdoc qtdiag qtpaths qtplugininfo; do
+		[[ ${PN} = ${x} ]] || qmakeargs+=( -no-feature-${x} )
+	done
+
+	# allow the ebuild to override what we set here
+	myqmakeargs=( "${qmakeargs[@]}" "${myqmakeargs[@]}" )
```

Comment:
maybe use another array (internal to the eclass) instead of hijacking `myqmakeargs`?

Reply:
hm, that means adding it to `qt5_qmake` just for qttools? not sure it is worth it, can do, but this is how it is done in kde5/ecm.eclass for years

Reply:
well, variables are cheap :) but the current way is perfectly fine with me too.